examples/example_rdf_import_export.py

```diff
@@ -0,0 +1,80 @@
+"""An example explaining the RDF import and export functionality."""
+
+# Please install the city ontology: $pico install city
+
+# If you did not install the city ontology
+# (pico install city),
+# you have to execute these commands first:
+# from osp.core import Parser
+# p = Parser()
+# p.parse("city")
+
+import uuid
+import re
+import os
+from rdflib import URIRef
+from osp.wrappers.sqlite import SqliteSession
+from osp.core.namespaces import city
+from osp.core.utils import serialize_rdf_graph, branch, import_rdf_file, \
+    pretty_print
+
+uuid_re = re.compile(r".*(http://www\.osp-core\.com/cuds#"
+                     r"([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}"
+                     r"-[a-z0-9]{4}-[a-z0-9]{12})).*")
+
+# Create CUDS structure
+c = branch(branch(city.City(name="Freiburg"),
+                  city.City(name="Pablo"),
+                  city.City(name="Yoav"),
+                  rel=city.hasInhabitant),
+           city.Neighborhood(name="Stühlinger"),
```

Comment:
The umlaut is making the execution fail for me at line 75 (pretty_print). Should not happen. Something is going on with the encoding of the exported file.

Reply:
Hmm interesting, it should be UTF-8... Maybe you're windows machine decides to use sth else, because we didn't explicitly specify the encoding?

Reply:
![image](https://user-images.githubusercontent.com/42069939/111583699-140e9200-87bd-11eb-9bd5-8475b093e8b3.png)

Reply:
I'll check if I can make it work on my machine.

Reply:
The statement `open()` is using the encoding `cp1252`, while rdflib expects `utf-8`. I just set the right encoding in the `open()` statements.